spotify/main.py

```diff
@@ -258,8 +284,8 @@ def main():
         # but we don't use it at the moment.
         # fetch_and_capture("episode_metadata", f"data/episodes/metadata/{id}", lambda: spotify_connector.episode_metadata(id, start, end), open_podcast_client, start, end)
 
-        start = dt.datetime.now() - dt.timedelta(days=3)
-        end = dt.datetime.now()
+        start = start_datetime - dt.timedelta(days=3)
```

Comment:
do we need to hardcode the days here, isn't it just the defined start date?

Reply:
right.